packages/libraries/cli/src/helpers/errors.ts

```diff
@@ -137,6 +137,16 @@ export class MissingCdnEndpointError extends HiveCLIError {
   }
 }
 
+export class InvalidTargetError extends HiveCLIError {
+  constructor() {
+    super(
+      ExitCode.BAD_INIT,
+      errorCode(ErrorCategory.GENERIC, 9),
```

Comment:
If this is a generic error (meaning it applies to multiple commands) then it'll be number 20. `errorCode(ErrorCategory.GENERIC, 20),` https://the-guild.dev/graphql/hive/docs/api-reference/cli#errors-119